data.yaml

```diff
@@ -190,6 +190,10 @@ categories:
         link: https://github.com/ValerySidorin/shclog
         description: Hclog adapter for slog.
 
+      - name: slog-aws-lambda
```

Comment:
I think this would fit better into the Enrichment category

Reply:
Good idea!